comparison_blogpost.txt

```diff
@@ -0,0 +1,59 @@
+Summary
+Overall, Apache Airflow is the heavier, more feature-dense and battle-tested service of the two, while Versatile Data Kit is more minimal and lightweight in comparison without sacrificing any fundamental capability and while offering a higher level of extensibility.
+
+Comparison points:
```

Comment:
I think we need a bit more research. 

I suggest you look at https://github.com/vmware/versatile-data-kit/wiki/Features and in https://airflow.apache.org/docs/apache-airflow/stable/index.html  and perhaps do a comparison table 

| Use case/feature | Airflow | VDK |
|---|---|---|
| xxx | yes,no.somewhat , and short explanation | yes,no,somewhat and short explanation |
| xxx |  |  |

And we need a picture (diagram) - a picture is 1000 words - which shows VDK and airflow together.  It's an option to reuse https://github.com/vmware/versatile-data-kit/tree/main/specs/vep-554-apache-airflow-integration#high-level-design or create your own

Reply:
I will also send you another link